### PR TITLE
Add the additional "worker_name" property to browserObject (fix #5)

### LINKED ISSUE #5: The “resolution” parameter isn’t used in the getWorkerName function.
https://www.devexpress.com/support/center/Question/Details/T341284

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Creates new BrowserStack workers and returns an array of workers ids.
 		* `browser`: The browser name
 		* `browser_version`: The browser version
 		* `device`: The device name
+		* `worker_name`: The worker name
```
Comment (AlexanderMoskovkin) [inline README.md]: I suppose we should check this description with a tech writer (maybe we should write a little more here or it's ok)

File: README.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ Creates new BrowserStack workers and returns an array of workers ids.
...
 * `callback`( `function( workerIds )` ): Invokes when when workers are created
```
Comment (MargaritaLoseva) [inline README.md]: Replace with:
"Invoked when workers are created"

File: README.md (hunk 1 above)
Comment (MargaritaLoseva) [inline README.md]: Let's write in this way: "The worker name that will be used by TestCafe to identify the browser for test execution."

**Comment by LavrovArtem** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
We need to publish new version I guess

**Comment by AlexanderMoskovkin** [comment]
>We need to publish new version I guess

@churkin please publish it. `testcafe-browserstack` is published from you `npm` account. BTW, We need to update the version before publishing

**Comment by churkin** [comment]
@AlexanderMoskovkin  done

**Pull request merged**